pkg/daemon/update.go

```diff
@@ -217,13 +219,62 @@ func addExtensionsRepo(osImageContentDir string) error {
 	return nil
 }
 
+// podmanRemove kills and removes a container
+func podmanRemove(cid string) {
+	// Ignore errors here
+	exec.Command("podman", "kill", cid).Run()
+	exec.Command("podman", "rm", "-f", cid).Run()
+}
+
+func podmanCopy(imgURL, osImageContentDir string) (err error) {
+	// make sure that osImageContentDir doesn't exist
+	os.RemoveAll(osImageContentDir)
+
+	// Workaround fixes for the environment where oc image extract fails https://bugzilla.redhat.com/show_bug.cgi?id=1862979
+	glog.Infof("Falling back to using podman cp to fetch OS image content")
+
+	// Pull the container image
+	var authArgs []string
+	if _, err := os.Stat(kubeletAuthFile); err == nil {
+		authArgs = append(authArgs, "--authfile", kubeletAuthFile)
+	}
+	args := []string{"pull", "-q"}
+	args = append(args, authArgs...)
+	args = append(args, imgURL)
+	_, err = pivotutils.RunExt(numRetriesNetCommands, "podman", args...)
+	if err != nil {
+		return
```

Comment:
this will return err?

Reply:
yes, it is named return https://tour.golang.org/basics/7